src/lib.rs

```diff
@@ -128,12 +128,18 @@ enum Fields {
 
 /// Defines a struct field.
 #[derive(Debug, Clone)]
-struct Field {
+pub struct Field {
     /// Field name
     name: String,
 
     /// Field type
     ty: Type,
+
+    /// Field documentation
+    documentation: String,
+
+    /// Field annotation
+    annotation: String,
```

Comment:
I think this probably should be a `Vec<String>` to support adding multiple annotations. In this case, the `annotation` builder function would push a new string.